coops/sentinels.tex

```diff
@@ -108,16 +108,16 @@ \subsection*{\MakeUppercase{Enemy Spawning\\and Movement}}
   \item Enemy armies spawn on a random Obelisk field (use hero miniatures from factions not in play).
   \item Determine which Obelisk field by rolling 2 Attack Dice. Apply the results as follows (D1/D2), reroll any 0:
   \begin{itemize}[leftmargin=15pt]
-    \item +1/+1 = North, +1/-1 = East
-    \item -1/+1 = West, -1/-1 = South
+    \item $+1$/$+1$ = North, $+1$/$-1$ = East
+    \item $-1$/$+1$ = West, $-1$/$-1$ = South
   \end{itemize}
   \item For a more balanced distribution, in Rounds 7 and 9, only roll the second Attack Die (D2). For D1, take the opposite result from what was rolled in the previous Round.
   Rounds 8 and 10 are entirely random.
   \item If the tile where an enemy is to spawn hasn't been discovered yet, flip it over and orient it as preferred.
   \item If there is a hero on the field where the enemy spawns, a fight starts immediately.
   \item Enemy armies move at the end of every turn, starting with the turn they spawned.
   \item Enemy armies have 3 MPs per round.
-  \item Enemy armies move as described in the rulebook (p. 33), but instead of capturing, they destroy everything in their path.
+  \item Enemy armies move as described in the rulebook (p.\,33), but instead of capturing, they destroy everything in their path.
```

Comment:
why the comma?

Reply:
It's not a comma. `\,` is a short non-breakable space. When the number is too far away from "p.", and even on the next line, it looks bad

Reply:
Hmm.. I'm wondering if this can be put to some macro if it makes sense to avoid future issues. Not mentioning that the `p.` will be different in translations

Reply:
Please do :) In the rulebook we already use `\,` via a macro. This PR is focused on typesetting